### DISCUSSION TITLE
Scientists vs. Engineers

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#543] [General]
The newscasts include lots of events that affect research, which refer to "scientists".  However I think what they really mean is engineers - it's _engineering_ teams you assign in R&D, not science teams.  I don't know though - some of these are more iffy.  "Rocket scientist", I think, is a popular term that includes engineers, and some of these could refer to scientists, such as the folks criticizing the space program.  And I'm uncertain what to do with "research scientists".  Thoughts, anyone?

OPERATION PAPERCLIP HAS BROUGHT IN A TOP GERMAN ROCKET SCIENTIST TO AID IN THE SPACE PROGRAM. R&D WILL IMPROVE FROM THIS DEVELOPMENT.
A GERMAN SCIENTIST IS RESCUED BY THE KGB AND WILL WORK FOR THE SUPREME SPACE PROGRAM. THERE WILL BE GREAT ADVANCES IN R&D.

INTELLIGENCE SOURCES TODAY HAVE REVEALED THAT THE KGB HAS CLEVERLY FOOLED THE CIA AND THE UNITED STATES' TOP SCIENTISTS WITH FALSE DATA. THE SAFETY FACTOR FOR THE e IS NOW LOWERED BY d.
THE EVIL CIA HAS CLEVERLY FOOLED THE KGB AND TOP SCIENTISTS WITH FALSE INFORMATION. THE SAFETY FACTOR FOR e IS NOW LOWERED BY d.

INTELLIGENCE SOURCES TODAY HAVE REVEALED THAT THE KGB HAS CLEVERLY FOOLED THE CIA AND THE UNITED STATES' TOP SCIENTISTS WITH FALSE DATA. THIS HAS DEGRADED THE SAFETY FACTOR OF THE e DOWN BY d.

THE GULAGS HAVE RE-EDUCATED SOME SCIENTISTS. THERE WILL BE IMPROVEMENTS THIS SEASON IN R&D.

A GERMAN ROCKET SCIENTIST 'VOLUNTEERS' ASSISTANCE TO THE GREAT SOVIET SPACE EFFORT. THIS WILL ASSIST IN R&D.

THE SPACE PROGRAM ANNOUNCED TODAY THAT IT HAS MANAGED TO RECRUIT A TOP SCIENTIST FROM A MAJOR CORPORATION. R&D WILL IMPROVE FROM THIS DEVELOPMENT.
THE SPACE PROGRAM HAS MANAGED TO TRANSFER A TOP SCIENTIST FROM AN EASTERN BLOC COUNTRY. R&D WILL IMPROVE THIS SEASON.

A TOP SCIENTIST HAS EMIGRATED FROM AN ALLIED COUNTRY TO WORK FOR THE SPACE PROGRAM. R&D WILL IMPROVE DUE TO THIS DEVELOPMENT.
INTERNAL SECURITY DELIVERS A TOP SCIENTIST TO WORK FOR THE SPACE PROGRAM. THIS WILL GREATLY AID R&D.

A FORMER GERMAN ROCKET SCIENTIST HAS ESCAPED FROM THE IRON CURTAIN. R&D WILL IMPROVE WITH THE AID OF THIS SCIENTIST.

A SOVIET SCIENTIST WHO WENT TO THE WEST HAS BEEN RESCUED BY SOVIET AGENTS AND IS BROUGHT BACK TO WORK FOR THE SPACE PROGRAM. R&D WILL ACCELERATE THIS SEASON.

A KEY SCIENTIST HAS ESCAPED FROM THE SOVIET UNION. INFORMATION FROM THIS SCIENTIST WILL AID IN R&D.
AN EX-NAZI SCIENTIST HAS BEEN EDUCATED AND EMBRACES THE COMMUNIST SYSTEM. THIS WILL ASSIST R&D THIS SEASON.

THE SPACE PROGRAM HAS LOST SEVERAL KEY RESEARCH SCIENTISTS IN A VERY SHORT PERIOD OF TIME. THIS SETBACK WILL MEAN WEAKER RESULTS IN R&D.
THE SOVIET SPACE PROGRAM HAS LOST SEVERAL KEY RESEARCH SCIENTISTS. THEY HAVE BEEN RELOCATED TO SIBERIA. THIS SETBACK WILL MEAN WEAKER RESULTS IN R&D THIS SEASON.

A CONTRACTOR FOR THE SPACE PROGRAM HAS GROSSLY UNDERESTIMATED THE DEVELOPMENT COST FOR THE e PROGRAM. R&D WILL COST AN ADDITIONAL 1 M.B. PER SCIENTIST TEAM PERMANENTLY.
THE CENTRAL COMMITTEE GROSSLY UNDERESTIMATED THE DEVELOPMENT COSTS FOR THE e PROGRAM. R&D COSTS AN ADDITIONAL 1 M.B. PER SCIENTIST TEAM PERMANENTLY.

A CONTRACTOR FOR THE SPACE PROGRAM HAS GROSSLY UNDERESTIMATED THE DEVELOPMENT COST FOR e PROGRAM. R&D WILL COST AN ADDITIONAL 1 M.B. PER SCIENTIST TEAM PERMANENTLY.
THE CENTRAL COMMITTEE GROSSLY UNDERESTIMATES THE DEVELOPMENT COSTS FOR THE e PROGRAM. R&D COSTS AN ADDITIONAL 1 M.B. PER SCIENTIST TEAM PERMANENTLY.

AMERICA'S TOP GERMAN SCIENTIST SPOKE TODAY ON THE ADVANTAGES OF THE 'EARTH ORBITAL RENDEZVOUS' APPROACH IN GETTING A MAN ON THE MOON. HE SPOKE OF THE LONG-RANGE ADVANTAGES OF 'EOR'.
A SCIENTIST FROM THE SOVIET ACADEMY OF SCIENCES SPOKE TODAY ON THE ADVANTAGES OF THE 'EARTH ORBITAL RENDEZVOUS' APPROACH IN GETTING A MAN ON THE MOON. HE SPOKE OF THE LONG-RANGE ADVANTAGES OF 'EOR'.

A TOP SCIENTIST CRITICIZED THE SPACE PROGRAM TODAY FOR THE HIGH COST OF MANNED EXPLORATION. ROBOT PLANETARY EXPLORERS WERE OFFERED AS A CHEAPER ALTERNATIVE.
A RIVAL SCIENTIST CRITICIZED THE SOVIET SPACE MINISTERS FOR THE HIGH COST OF MANNED EXPLORATION. ROBOT PLANETARY EXPLORERS WERE OFFERED AS A CHEAPER ALTERNATIVE.

A SOVIET SCIENTIST HAS SUBMITTED A PROPOSAL FOR A LUNAR HABITAT CAPABLE OF SUSTAINING COSMONAUTS FOR UP TO THIRTY DAYS ON THE MOON.

**Comment by hweimer** [comment]
In some sense, research means scientists and development means engineers, so we have both involved. I would leave things as they are as this is a pretty grey area anyway.

**Reply by peyre** [reply]
True, I noticed kind of halfway through there that it's a grayer area than I originally thought.  I do wonder, though, if it might be worth changing the ones that specifically say R&D will cost more per scientist team, to line up with the manual and the help file, which call those engineering teams.

I guess I'm a little sensitive about the scientist vs. engineering thing because of all the nonsense that went on in the 2000s with the Intelligent Design movement.  Oftentimes engineers, or medical doctors, would come on forums claiming to be scientists, and _they_ didn't think there was anything to evolution.  (Turns out it's surprisingly common for engineers and physicians to be creationists - who knew?)

And on a funnier note, I think xkcd pointed out that most "mad scientists" are actually mad _engineers_.  lol

Edit: ah, no, not xkcd.  I see it's from a different online comic altogether.  [https://www.evilmadscientist.com/2015/evil-mad-engineers/](url)
